lib/mshoplib/setup/TablesUpdateCharsetCollation.php

```diff
@@ -171,6 +171,40 @@ protected function checkColumns( \Aimeos\MW\Setup\DBSchema\Iface $schema, $table
 			return true;
 		}
 
+		return false;
+	}
+
+    /**
+	 * Checking the MySql compatibility.
+	 *
+	 * @param \Aimeos\MW\Setup\DBSchema\Iface $schema
+	 * @return bool
+	 */
+	protected function checkMySqlCompatibility( \Aimeos\MW\Setup\DBSchema\Iface $schema )
+	{
+		if ( !$schema instanceof \Aimeos\MW\Setup\DBSchema\Mysql )
+		{
+			return true;
+		}
+
+		// Ask the database for it's type and version.
+		// A MariaDB gets identified as a MySql 5.5.5 by doctrine.
+		// We can not rely on it to tell us the actual version and ask
+		// the server directly.
+		$conn = $this->acquire();
```

Comment:
You can use getValue() instead: https://github.com/aimeos/aimeos-core/blob/master/lib/mwlib/src/MW/Setup/Task/Base.php#L197

```
$version = $this->getValue( 'SELECT version() AS "version"', 'version', 'db-customer' );

Reply:
ummm . . . 
The getValue method uses `$this->getConnection` which is marked as deprecated, so I opted for `acquire()` and `release()`.

Shall I change it?

Reply:
That has been fixed in the master branch and it's not using getConnection() any more